src/js/controllers/confirm.js

```diff
@@ -268,6 +282,8 @@ angular.module('copayApp.controllers').controller('confirmController', function(
       $scope.getSendMaxInfo();
     } else
       setWallet(wallet);
+
+    applyButtonText(wallet.credentials.n > 1);
```

Comment:
`n` is number of copayers. `m` is the number of required signature.

I think that it should be `m`.